buildSrc/src/main/groovy/com/sun/javafx/gradle/NativeCompileTask.groovy

```diff
@@ -102,7 +102,7 @@ class NativeCompileTask extends DefaultTask {
         updateFiles();
         def source = project.files(allFiles);
         boolean forceCompile = false;
-        final Set<File> files = new HashSet<File>();
+        Set<File> files = new HashSet<File>();
```

Comment:
I'm not sure that this code is in error. The `files` variable is a set and is assigned only once in the above declaration. Elements are added to the `files` set, but that can still be done even if `files` is final.